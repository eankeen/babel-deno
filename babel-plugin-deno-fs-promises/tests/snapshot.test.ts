import * as babel from '@babel/core'

import { readTranspileAndWrite } from './test.utils'

test('first test', async () => {
  const {
    denoResult
  } = await readTranspileAndWrite('first')


  expect(denoResult.code).toMatchSnapshot()
  expect(denoResult.ast).toMatchSnapshot()
})
