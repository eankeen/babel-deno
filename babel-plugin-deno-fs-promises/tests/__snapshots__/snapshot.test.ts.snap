// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first test 1`] = `
"(async () => {
  const contents = await Deno.readTextFile('./file', {
    encoding: 'utf8'
  });
  console.log(contents);
})();"
`;

exports[`first test 2`] = `null`;
